src/lua/test.lua

```diff
@@ -1116,11 +1131,11 @@ function _Gtme.executeTests(package, fileName)
 	end
 
 	if ut.files_created == 1 then
-		printError("One file was created along the tests.")
+		printError("One file or directory was created along the tests.")
 	elseif ut.files_created > 1 then
-		printError(ut.files_created.." files were created along the tests.")
+		printError(ut.files_created.." files and/or directories were created along the tests.")
 	else
-		printNote("No file was created along the tests.")
+		printNote("No file or directory was created along the tests.")
```

Comment:
You will need to update all the other logs related to tests with these changes in the report.